src/data/set6/champions.ts

```diff
@@ -53,6 +53,22 @@ export default {
 		},
 	},
 
+	[ChampionKey.Kassadin]: {
+		cast: (elapsedMS, spell, champion) => {
+			const target = champion.target
+			if (!target) { return console.log('No target', champion.name, champion.team) }
+			champion.queueProjectile(elapsedMS, spell, {
+				spell,
+				target,
+				collidesWith: champion.opposingTeam(),
+				destroysOnCollision: true,
+				onCollision: () => {
+					target.bonuses.push([SpellKey.Mana, [[BonusKey.ManaReductionPercent, -0.5]]]) // TO DO: only let this "bonus" hold for one cast
```

Comment:
We need a TODO or implementation for `DamageReduction`. We can make it expire by including a third parameter in the BonusVariable: `[BonusKey.DamageReduction, 25%, elapsedMS + 5s]` (values should be pulled from the spell).

Reply:
have tentatively added the cast time to the `DamageReduction ` bonus.